src/translations/translations.en.js

```diff
@@ -195,6 +195,13 @@ export const translations = {
   'cc-datetime-relative.distance': ({ date }) => formatDistanceToNow(date),
   'cc-datetime-relative.title': ({ date }) => formatDate(lang, date),
   //#endregion
+  //#region cc-doc-card
+  'cc-doc-card.link': ({ link, product }) => sanitize(`<a href=${link} aria-label="Read the ${product} documentation">Read the documentation</a>`),
```

Comment:
the `aria-label` value must start with the visible text (word for word):
"Read the documentation - ${product}"